source/_integrations/tplink.markdown

```diff
@@ -42,24 +44,24 @@ There is currently support for the following device types within Home Assistant:
 - HS103
 - HS105
 - HS110 (supports consumption sensors)
-- KP105
+- KP105 (is not mentioned in the TP-Link FAQ as working/not working)
 - KP115 (supports consumption sensors)
 
 ### Strip (Multi-Plug)
 
-- HS107 (indoor 2-outlet)
-- HS300 (powerstrip 6-outlet) (supports consumption sensors)
-- KP303 (powerstrip 3-outlet)
-- KP400 (outdoor 2-outlet)
+<!-- - HS107 (indoor 2-outlet) # Does not support local control according to TP-Link FAQ (as of date 2022-04-25) -->
```

Comment:
Not worth changing these as these are based on what users have reported to be functioning even if tplink's docs / official stance disagree.

Reply:
Alright, I made a table to reflect both infos to give the users a chance to get an overview :)

Thanks!